Source/Urho3D/Graphics/DebugRenderer.cpp

```diff
@@ -505,7 +541,7 @@ void DebugRenderer::Render()
     graphics->SetBlendMode(lineAntiAlias_ ? BLEND_ALPHA : BLEND_REPLACE);
     graphics->SetColorWrite(true);
     graphics->SetCullMode(CULL_NONE);
-    graphics->SetDepthWrite(true);

```

Comment:
> If there's a lot of overlapping lines, it's true the depth writing is a rather important part of debug rendering.

I see AddTriangle method. So somebody may use DebugRenderer for drawing some solid stuff. So it may be important to draw such solid triangles with propper depth sorting.